packages/editor/src/browser/editor-preferences.ts

```diff
@@ -240,7 +282,12 @@ export const editorPreferenceSchema: PreferenceSchema = {
             'description': 'Accept suggestions on provider defined characters.'
         },
         'editor.snippetSuggestions': {
-            'enum': ['top', 'bottom', 'inline', 'none'],
+            'enum': [
+                'top',
+                'bottom',
+                'inline',
```

Comment:
In the other enums, yo u usually put the default value the first entry of the enum